src/common/Connection.ts

```diff
@@ -67,8 +67,8 @@ export class Connection extends EventEmitter {
   }
 
   /**
-   * @param {string} sessionId
-   * @returns {?CDPSession}
+   * @param sessionId - Gets Session ID
+   * @returns - Gets CDPSession
```

Comment:
You don't need a dash after `@returns` - see https://tsdoc.org/pages/tags/returns/

So this should be

```
@returns the current CDP session if it exists